packages/hooks/src/useSize/index.ts

```diff
@@ -9,7 +9,7 @@ type Size = { width: number; height: number };
 function useSize(target: BasicTarget): Size | undefined {
   const el = getTargetElement(target);
   const [state, setState] = useRafState<Size | undefined>(
-    el ? { width: el.clientWidth, height: el.clientHeight } : undefined,
+    () => el ? { width: el.clientWidth, height: el.clientHeight } : undefined,
   );
 
```

Comment:
What about put `const el = getTargetElement(target);` into `useRafState(`'s callback, just like:

```ts
const [state, setState] = useRafState<Size | undefined>(() => {
  const el = getTargetElement(target);

  return el ? { width: el.clientWidth, height: el.clientHeight } : undefined;
});
```

So `const el = getTargetElement(target);` will be executed only on the initial render.

Reply:
Yep, it seems that the size only need to be initialized when target was already exist.